src/current/_includes/v23.1/sidebar-data/schema-design.json

```diff
@@ -0,0 +1,135 @@
+{
+    "title": "Schema Design",
+    "items": [
+      {
+        "title": "Overview",
```

Comment:
The page this links to is not an overview of schema design. It's an overview of database schemas. Suggest changing the name to "Database Schemas". 

I'd like to see us include an overview page on CockroachDB Schema Design that includes a treatment of the topics here as part of a later docs project:
https://docs.google.com/document/d/1obWDUJEm1M8cOaLyl5RxXeqBqndtuwJX1I8tjo4a00o/edit#heading=h.yx60tat7xy8

Reply:
To be clear, this section of the docs is about schema design, it's just this page that is only about database schemas.

Reply:
Updated the nav title and will file an issue to create a Schema Design Overview page in the future